src/EventListener/Pimcore/ChangeListener.php

```diff
@@ -37,7 +38,14 @@ public function handle(AssetEvent|DataObjectEvent|DocumentEvent $event): void
             return;
         }
 
-        $this->propagateChanges->handle($this->getFreshElement($event->getElement()));
+        $element = $event->getElement();
+
+        // If a folder is created in the assets section in Pimcore 11 the type is set to Unknown.
+        if ($element instanceof Unknown) {
```

Comment:
The comment implies a stronger condition. Unfortunately, I couldn't find a clean way to check whether we're handling something that's being *created* but we could check whether it's indeed a folder (and not an asset of unknown type):

```suggestion
        if ($element instanceof Unknown && $element->getType() === 'folder') {
```